### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by Ayush7614**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -497,4 +497,11 @@
     "imageUrl": "https://user-images.githubusercontent.com/66197682/105803643-eaf33180-5fc3-11eb-8f97-42de7cee0f05.png",
     "issueNumber": 279
   }
+    {
+    "name": "Ayush Kumar",
+    "githubUsername": "Ayush7614",
+    "imageUrl": "https://user-images.githubusercontent.com/67006255/106144942-ee8ce100-619a-11eb-908d-4bdceade5661.png",
+    "issueNumber": 283
+  },
```
Comment (eddiejaoude) [inline data.json:505]: the trailing comma is not needed

File: data.json [hunk 2]
```diff
@@ -497,4 +497,11 @@
     "imageUrl": "https://user-images.githubusercontent.com/66197682/105803643-eaf33180-5fc3-11eb-8f97-42de7cee0f05.png",
     "issueNumber": 279
   }
+    {
```
Comment (eddiejaoude) [inline data.json:500]: Missing comma between objects

**Comment by eddiejaoude** [comment]
Thanks @Ayush7614 , I left a couple of inline comments, any questions let us know 🤓

**Comment by Ayush7614** [comment]
why my screenshot not seen in README.md

**Comment by eddiejaoude** [comment]
Because these changes have not been merged yet

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**